src/components/base/RetryButton.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import Button from 'components/base/Button'
+import { translate } from 'react-i18next'
+
+import type { T } from 'types/common'
+
+type Props = {
+  t: T,
+  onClick: Function,
+  event?: string,
+}
+
+class RetryButton extends PureComponent<Props> {
+  render() {
+    const { t, onClick, event, ...props } = this.props
+    return (
+      <Button onClick={onClick} {...props} event={event}>
```

Comment:
both event and onClick can be left as part of `props` and no need to define neither in Props type